src/core/account/account-cleaners.js

```diff
@@ -58,3 +73,25 @@ export const asPluginSettingsFile = asObject({
   // Swap plugins:
   swapSettings: asOptional(asMap(asSwapSettings), {})
 }).withRest
+
+/**
+ * The settings file managed by the GUI.
+ */
+export const asGuiSettingsFile = asObject({
+  customTokens: asArray(
+    asObject({
+      contractAddress: asString,
+      currencyCode: asString,
+      currencyName: asString,
+      denomination: asString,
+      denominations: asArray(asEdgeDenomination),
+      isVisible: asOptional(asBoolean, true),
+      multiplier: asString,
+      walletType: asOptional(asString, 'wallet:ethereum')
+    })
+  )
+})
+
+export const asCustomTokensFile = asObject({
+  customTokens: asMap(asMap(asEdgeToken))
```

Comment:
Hasn't `asMap` been replaced in favor of `asObject`?

Reply:
Derp. I'll fix this.